tests/csrf_tests/tests.py

```diff
@@ -655,7 +679,9 @@ def test_no_session_on_request(self):
             'SessionMiddleware must appear before CsrfViewMiddleware in MIDDLEWARE.'
         )
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
-            CsrfViewMiddleware().process_view(HttpRequest(), None, (), {})
+            req = HttpRequest()
```

Comment:
Move the first two lines out of the context manager so it's more clear which line is raising the exception.